### ISSUE TITLE
storage.porter.filesystem error on 0.27.1 with windows

### ISSUE THREAD

**Issue by CtrlDot** [deislabs/porter#1154]
## Describe the bug
Get the following error message when running porter parameters generate with a fresh install of v0.27.1.

Error: could not load storage plugin: could not connect to the storage.porter.filesystem plugin: fork/exec..... parameter is incorrect

## To Reproduce
1. Download version 0.27.1
2. porter parameters list

## Expected behavior
A list of the parameter sets already created

## Porter Command and Output
```
Error: could not load storage plugin: could not connect to the storage.porter.filesystem plugin: fork/exec..... parameter is incorrect
```

## Version
v0.27.1

**Comment by carolynvs** [comment]
Thanks for the report! I am able to reproduce the bug on my test machine.

**Comment by jasmdk** [comment]
I have the same issue with 0.28.0

**Comment by carolynvs** [comment]
Thank you for reporting (and re-reporting) this problem. It has been fixed in [v0.28.1](https://github.com/deislabs/porter/releases/tag/v0.28.1) which was just realized a few minutes ago.

Please let us know if you are still experiencing any problems!
